units/console-getty.service.in

```diff
@@ -23,10 +23,11 @@ ConditionPathExists=/dev/console
 # The '-o' option value tells agetty to replace 'login' arguments with an
 # option to preserve environment (-p), followed by '--' for safety, and then
 # the entered username.
-ExecStart=-/sbin/agetty -o '-p -- \\u' --noclear --keep-baud console 115200,38400,9600 $TERM
+ExecStart=-/sbin/agetty -o '-p -- \\u' --noclear --keep-baud - 115200,38400,9600 $TERM
 Type=idle
 Restart=always
 UtmpIdentifier=cons
+StandardInput=tty
```

Comment:
hmm, this should set StandardOutput=tty too, no?

(StandardError=tty is then implied and can be omitted)

Reply:
`StandardInput=tty` is sufficient for agetty, it dups it over to stdout and stderr itself so not sure if we need to do it here as well.

Reply:
hmm, it does that? isn't that a weird thing to do? is this documented?

kinda appears to me we should do this explicitly here. Since it's quite surprising behaviour I would say. (or to say this differently: it should have a comment at the very least, explaining why this isn't needed, but i think an explicit setting is nicer than a comment...

Reply:
Added it, everything seems to work, doesn't seem like agetty documents that it dups stdin over stdout/stderr but checked the source to verify. Anyway, this works so I agree there's no reason not to specify it.